docs/source/tags/labels.md

```diff
@@ -35,3 +35,31 @@ Use with the following data types: audio, text. Other data types have type-speci
   <Text name="txt-1" value="$text" />
 </View>
 ```
+**Example** *(This config with dynamic labels)*  
```

Comment:
Revise `labels.md` with the following 
---
title: Labels
type: tags
order: 411
meta_title: Labels Tag for Labeling Regions
meta_description: Customize Label Studio by using the Labels tag to provide a set of labels for labeling regions in tasks for machine learning and data science projects.
---

The Labels tag provides a set of labels for labeling regions in tasks for machine learning and data science projects. Use the Labels tag to create a set of labels that can be assigned to identified region and specify the values of labels to assign to regions.

All types of `Labels` can have dynamic `value` to load labels from task. This task data should contain a list of options to create underlying `<Label>`s. All the parameters from options will be transferred to corresponding tags.

The Labels tag supports the following data types for both audio and text. Other data types have type-specific Labels tags.


### Parameters

| Param | Type | Default | Description |
| --- | --- | --- | --- |
| name | <code>string</code> |  | Name of the element |
| toName | <code>string</code> |  | Name of the element that you want to label |
| [choice] | <code>single</code> \| <code>multiple</code> | <code>single</code> | Configure whether you can select one or multiple labels for a region |
| [maxUsages] | <code>number</code> |  | Maximum number of times a label can be used per task |
| [showInline] | <code>boolean</code> | <code>true</code> | Whether to show labels in the same visual line |
| [opacity] | <code>float</code> | <code>0.6</code> | Opacity of rectangle highlighting the label |
| [fillColor] | <code>string</code> |  | Rectangle fill color in hexadecimal |
| [strokeColor] | <code>string</code> | <code>&quot;#f48a42&quot;</code> | Stroke color in hexadecimal |
| [strokeWidth] | <code>number</code> | <code>1</code> | Width of the stroke |

### Example

Basic labeling configuration to apply labels to a passage of text:
```html
<View>
  <Labels name="type" toName="txt-1">
    <Label alias="B" value="Brand" />
    <Label alias="P" value="Product" />
  </Labels>
  <Text name="txt-1" value="$text" />
</View>
```
**Configure using Dynamic Labels**
*Example 1* 
```html
<PolygonLabels name="product" toName="shelf" value="$labels" />
<!-- {
  "data": {
    "options": [
      { "value": "Big brand" },
      { "value": "Another brand", "background": "orange" },
      { "value": "Local brand" },
      { "value": "Green brand", "alias": "Eco", showalias: true }
    ]
  }
} -->
```
*Example 2*   
```html
<PolygonLabels name="product" toName="shelf">
  <Label value="Big brand" />
  <Label value="Another brand" background="orange" />
  <Label value="Local brand" />
  <Label value="Green brand" alias="Eco" showAlias="true" />
</PolygonLabels>
```